### PR TITLE
Fixed Truncating Text in Team Invite URL

### LINKED ISSUE #7991: Remove truncation and allow horizontal scroll through the entire team invitation URL
### Bug description

When a user tries to invite other team members they are presented with a modal that includes an invitation URL.

While there's a shortcut button to copy the invitation URl to the clipboard, the user can horizontally scroll on the disabled text input.

| 1 | 2 |
|-|-|
| invitation-url-1 | invitation-url-2 |

### Steps to reproduce

1. Create a team
2. Invite members
3. Scroll horizontally on the disabled text input containing the invitaiton URL

### Workspace affected

_No response_

### Expected behavior

Do not truncate the invitation URL but allow users to scroll horizontally through the entire link text.

### Example repository

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Harshil-Jani**
## Description

This PR is associated for making the Invite Members Input field as scroll-able which will avoid truncating the URLs.

## Related Issue(s)

Fixes #7991 

## How to test

- Create a team
- Invite members
- Scroll horizontally on the disabled text input containing the invitaiton URL 
## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
@Harshil-Jani: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **laushinka** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @laushinka` in a comment when ready.

Associated issue: *#7991*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**
- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/dashboard/src/teams/Members.tsx
Comment (laushinka) [inline components/dashboard/src/teams/Members.tsx:209]: Thanks for the contribution! I think you'll need to remove the `truncate` here?

Comment (Harshil-Jani) [reply components/dashboard/src/teams/Members.tsx:209]: > I think you'll need to remove the `truncate` here?

Oh Yes. We have to. Missed out on it. Let me change this.

**Review by gtsiolis — APPROVED** [review]
Thanks for contributing, @Harshil-Jani! 🧡 

UX changes look good! ✔️ 

You'll also need to sign a Contributor License Agreement (CLA)[[1](https://en.wikipedia.org/wiki/Contributor_License_Agreement)] once before merging your first contribution. Looping in @meysholdt to reach out about the CLA. 🏓

**Comment by Harshil-Jani** [comment]
@meysholdt  Any updates for further processes regarding CLA ?

**Comment by Harshil-Jani** [comment]
@meysholdt @gtsiolis Can you please tell about CLA ?

**Comment by meysholdt** [comment]
hi @Harshil-Jani! 

Thank you for your contribution and apologies for the late response. I've sent our CLA to you via email/DocuSign. Feel free to reach out via moritz@gitpod.io if you have questions. 

regards,
  Moritz

**Comment by Harshil-Jani** [comment]
> hi @Harshil-Jani!
> 
> Thank you for your contribution and apologies for the late response. I've sent our CLA to you via email/DocuSign. Feel free to reach out via [moritz@gitpod.io](mailto:moritz@gitpod.io) if you have questions.
> 
> regards, Moritz

Yo ! Signed CLA. Thankyou for reaching out.

**Comment by laushinka** [comment]
Hi @Harshil-Jani, really sorry for how long this PR is taking 🙏🏽🙏🏽🙏🏽 Once you squash the commits, this will be approved!

**Review by gtsiolis — CHANGES_REQUESTED** [review]
@Harshil-Jani could you squash the commit in this PR as @laushinka mentioned in https://github.com/gitpod-io/gitpod/pull/8044#issuecomment-1046294634?

**Pull request closed without merging**
